components/prism-ignore.js

```diff
@@ -0,0 +1,34 @@
+(function (Prism) {
+	Prism.languages.ignore = {
+		// https://git-scm.com/docs/gitignore
+		'header': {
+			pattern: /(^|[^\\])###.*/,
```

Comment:
Since the whole format is line-based, we can simply require a header to start at the start of a line. That way, we don't have to check for any backslashes and we won't have problems with comments like this:

```
# comment ### foo
```

Please use the same trick for the `section` and `comment` patterns as well.

```suggestion
			pattern: /^###.*/m,
```